exit.go

```diff
@@ -151,19 +153,31 @@ func (te *TunaExit) handleSession(session *smux.Session) {
 				continue
 			}
 			cost += entryToExitPrice*common.Fixed64(in)/TrafficUnit + exitToEntryPrice*common.Fixed64(out)/TrafficUnit
+			totalBytes += common.Fixed64(in) + common.Fixed64(out)
 		}
-		return cost
+		return cost, totalBytes
 	}
 
 	if !te.config.Reverse {
 		npc, err = te.Wallet.NewNanoPayClaimer(te.config.BeneficiaryAddr, int32(claimInterval/time.Millisecond), onErr)
 		if err != nil {
 			log.Fatalln(err)
 		}
+		getTotalTraffic := func() common.Fixed64 {
```

Comment:
Why not use `getTotalCost` as `getTotalTraffic` since it also returns total traffic?